lgsm/functions/core_steamcmd.sh

```diff
@@ -0,0 +1,332 @@
+#!/bin/bash
+# LinuxGSM core_steamcmd.sh function
+# Author: Daniel Gibbs
+# Website: https://linuxgsm.com
+# Description: Core functions for SteamCMD
+
+functionselfname="$(basename "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+fn_install_steamcmd(){
+	if [ "${shortname}" == "ark" ]&&[ "${installsteamcmd}" == "1" ]; then
+		steamcmddir="${serverfiles}/Engine/Binaries/ThirdParty/SteamCMD/Linux"
+	fi
+	if [ ! -d "${steamcmddir}" ]; then
+		mkdir -p "${steamcmddir}"
+	fi
+	remote_fileurl="${1}"
```

Comment:
@dgibbs64 do you know if these are needed at all here ?
There is a open issue on shellcheck side.

Reply:
I can't see any shellcheck issues relating to this if statement